.husky/commit-msg

```diff
@@ -1,10 +1,8 @@
-#!/bin/sh
-
-FILE_NAME="$(dirname "$0")/_/husky.sh"
-
 # Conditionally invoke so as to avoid running pnpm commands if this is a clean checkout (ie before installing node modules)
-if [ -f $FILE_NAME ]; then
-  . $FILE_NAME
-
-  npx --no-install commitlint --edit "$1"
+if ! [ -d node_modules ]; then
+  echo "Skipping $0 hook as node modules are not installed" >&2
+  exit 0
 fi
+
+echo "" # empty line to help reading
```

Comment:
Since there are lots of line being written down by the test output, it might be worth printing something more explicit to show what the following output is about. Since each step happening during `pnpm prepare-code` &co is displayed by showing two empty lines followed by a line starting with `> `, I suggest we do the same here to announce this step and don't just print an empty line, giving something like 
```


> checking commit message with `commitlint`
```
or similar. What do you think?

Reply:
```suggestion

echo ''
echo '> checking commit message with `commitlint`'
```

It's okay for me. I only added one newline because the first one is already created (I assume by as a space after the last command).